src/flask_cognito_lib/config.py

```diff
@@ -48,12 +48,28 @@ class Config:
     CONTEXT_KEY_COGNITO_SERVICE = "aws_cognito_service"
     CONTEXT_KEY_TOKEN_SERVICE = "aws_jwt_service"
     COOKIE_NAME = "cognito_access_token"
+    COOKIE_NAME_REFRESH = "cognito_refresh_token"
 
     @property
     def disabled(self) -> bool:
         """Return True if Cognito Authentication is disabled"""
         return get("AWS_COGNITO_DISABLED", required=False, default=False)
 
+    @property
+    def refresh_flow_enabled(self) -> bool:
+        """Return True if Cognito Refresh flow is enabled"""
+        return get("AWS_COGNITO_REFRESH_FLOW_ENABLED", required=False, default=False)
+
+    @property
+    def refresh_cookie_encrypted(self) -> bool:
+        """Return True if Cognito Refresh cookie should be encrypted"""
+        return get("AWS_COGNITO_REFRESH_COOKIE_ENCRYPTED", required=False, default=True)
```

Comment:
Can you add these to the docs as well please? (`docs/config.md`)

Reply:
Sure, once implementation is finalized I'll update the docs respectively